### PR TITLE
New icons for Mind Frayer and BSDM

### PR DESCRIPTION
**Pull request by infraredbaron**
## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:
add: New Icons for Mind Frayer and BSDM + in-hands for a box
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/storage/bsdm.dm [hunk 1]
```diff
@@ -10,11 +11,13 @@
...
 	owner = user.mind
 	to_chat(user, "You claim \the [src].")
-
+	icon_state = "bsdm_sealed"
+	
 /obj/item/weapon/storage/bsdm/verb/activate()
 	set name = "Launch"
 	set category = "Object"
 	set src in oview(1)
+	icon_state = "bsdm_teleporting"
```
Comment (mattroks101) [inline code/game/objects/items/weapons/storage/bsdm.dm]: Does this need to return to it's original state?

**Comment by skraaak** [comment]
There is one loose white pixel in the last frame of the BDSM_Sealing animation.

**Comment by infraredbaron** [comment]
I'm pretty confident that github having some issues on preview, because there is no rogue pixels at all in my version. Nor local nor here

**Comment by Stealthkibblers** [comment]
no new sprites for spying sensors? I think they need it the most.

**Pull request merged**